prefixed_logger.go

```diff
@@ -0,0 +1,140 @@
+package log
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/neighborly/go-errors"
+	"github.com/sirupsen/logrus"
+)
+
+type PrefixedLogger struct {
+	Prefix string
+}
+
+func NewPrefixedLogger(prefix string) PrefixedLogger {
```

Comment:
You might actually require a Logger instance as an argument to the constructor rather than assuming `logger` is initialized. Client can use `GetLogger()` to retrieve the instance to pass to the constuctor.